tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -125,9 +191,77 @@ private bool WriteAndEndResponse()
         httpResponse.Flush();
         httpResponse.Close();
         _context.ApplicationInstance.CompleteRequest();
+
         return true;
     }
 
+    private bool TryCreateThrowHttpResponseExceptionDynMeth(out Action<string, string> throwHttpResponseException)
+    {
+        throwHttpResponseException = null;
+
+        try
+        {
+            var exceptionType = Type.GetType("System.Web.Http.HttpResponseException, System.Web.Http");
```

Comment:
This method seems non critical, but WDYT about caching all these Reflection calls so successive calls are quicker?

Reply:
The dynamic method that ends up being created is already cached, so we shouldn't need to call this method more than once (or once per appdomain at least).